frontend/packages/console-app/src/actions/creators/common-factory.ts

```diff
@@ -16,6 +16,8 @@ export type ResourceActionCreator = (
   obj: K8sResourceKind,
   relatedResource?: K8sResourceKind,
   message?: JSX.Element,
+  isGeneratedByConsole?: boolean,
+  isKnativeResource?: boolean,
```

Comment:
We should try to avoid adding specific resource (knative) to so generic functions like the `CommonActionFactory`.

Couldn't we override the Delete action in a smarter way? Maybe with an action extension?